pkg/testcore/suites/functional-suites.go

```diff
@@ -886,9 +886,6 @@ func (cts *CapacityTrackingSuite) Run(ctx context.Context, storageClass string,
 	// POD should stay in pending state if capacity is zero
 	log.Infof("Updating CSIStorageCapacity for %s storage class, setting capacity to %s", color.YellowString(storageClass), color.HiYellowString("%d", 0))
 	capacities, _ := clients.CSISCClient.GetByStorageClass(ctx, storageClass)
```

Comment:
```suggestion
	capacities, err := clients.CSISCClient.GetByStorageClass(ctx, storageClass)
```

Please make this change and keep the err check after this line

Reply:
sure